1.12 - Avoid Detection - How To - Automate Logging On/Off

rhyys5 opened the pull request:
Hello all,

I was using the bot, works great! However, I was watching movies while fishing when a player appeared next to me (obviously a GM) and teleported me across the lake. I started moving around and the player left. If I was away from the computer at that time I probably would have been banned. So, I decided to create a batch file to accomplish these goals.

Log into the game
Start Fishing
Fish for 10 minutes (Modifiable)
Log off of the game
Attached is the .bat and .vbs file:

NOTES:

You will need to modify drives, paths, login, and START/STOP fishing key info in the 2 files

The "TIMEOUT /T 600" in the .bat file determines how long you fish before it logs you off

Just schedule the .bat file to run as often as you like. Example, if you set the .bat file to run every 45 minutes, you will fish for 10 minutes, and be logged off for 35 minutes until you start fishing again. This should make you look less suspicious. Hope you find this helpful!!!

The TIMEOUT /T 600 in the .bat file determines how long you fish before it logs you off

[Fishing.bat.txt](https://github.com/UltimateFishbot/UltimateFishbot/files/2659959/Fishing.bat.txt)
[fishing.vbs.txt](https://github.com/UltimateFishbot/UltimateFishbot/files/2659960/fishing.vbs.txt)

OverlordQ committed:
Create .travis.yml

OverlordQ committed:
Update .travis.yml

brian-man committed:
build in VS2017

brian-man committed:
add BobberLocation memory to Eyes

brian-man committed:
add 1s delay after Jump movement

brian-man committed:
rename NotEared to NotHeard in FishingStats

brian-man committed:
add NotLooted to FishingStats

brian-man committed:
update Deutsch localization of Total

brian-man committed:
reorder frmStats.Designer.cs to match UI

brian-man committed:
standardize colon positioning in UI

brian-man committed:
add random delay between splash and loot

brian-man committed:
GetNoFishCursor now respects ScanningDelay

brian-man committed:
Eyes.LookForBobber now checks current location first

brian-man committed:
GetNoFishCursor now uses GetCurrentCursor

brian-man committed:
GetNoFishCursor is now called only once per Start-Stop session

brian-man committed:
add MILLISECONDS_PER_ to MINUTE and SECOND constant names

brian-man committed:
ResetTimers now uses FishWait instead of hardcoded 22

brian-man committed:
Manager.Fish now listens while scanning for bobber

brian-man committed:
remove post-splash delay if splash occurs before bobber is found

brian-man committed:
frmMain is now a topmost window

brian-man committed:
fix line breaks and text indenting in *.Designer.cs

daniel-widrick committed:
Merge pull request #1 from daniel-widrick/issue-#91

Issue #91

daniel-widrick committed:
Merge of Brian-Man updates

daniel-widrick committed:
Merge pull request #3 from daniel-widrick/brian-man-brian-man

Merge conflict resolution for Brian Man changes

daniel-widrick committed:
Update to persistent functions

daniel-widrick committed:
updated readme

daniel-widrick committed:
Merge pull request #4 from daniel-widrick/readme.md

updated readme

daniel-widrick committed:
Merge pull request #5 from OverlordQ/CSharp

merge of travis settings

daniel-widrick committed:
Merge pull request #6 from daniel-widrick/brian-man-brian-man

Brian-man's contributions

daniel-widrick committed:
reset the day's playing. that was all a majot mess

daniel-widrick committed:
increase check speeds

daniel-widrick committed:
don't search for window on loot

Commit:
spanish translation courtesy of edu5ardo

Commit:
Hungarian translation, logging feature added, safety check url fixed, Ear avg improved

daniel-widrick committed:
Merge pull request #8 from Szabka/CSharp

Hungarian translation, logging feature added, safety check url fixed,…

Commit:
late bait, cursor capture, cursor recheck, parallel hear and cursorfind

Commit:
late bait, cursor capture, cursor recheck, parallel hear and cursorfind

Commit:
Fishing stops after 10 fail in row, bobber position cache

Commit:
Fishing stops after 10 fail in row, bobber position cache

daniel-widrick committed:
Merge pull request #9 from daniel-widrick/español

Español & hungarian

daniel-widrick committed:
Merge pull request #10 from Szabka/CSharp

Szabka work

Commit:
unused method deleted in win32, wowwindow ptr refreshed at fishing start.

Commit:
unused method deleted in win32, wowwindow ptr refreshed at fishing start.

Commit:
Epic bobber detection via image analysis
Inspired by https://www.codeproject.com/Articles/10248/Motion-Detection-Algorithms

Commit:
Epic bobber detection via image analysis
Inspired by https://www.codeproject.com/Articles/10248/Motion-Detection-Algorithms

Commit:
Code style fix, prepare for release

Commit:
Code style fix, prepare for release

daniel-widrick committed:
Merge pull request #12 from Szabka/CSharp

Changes from Szabka

daniel-widrick committed:
Merge pull request #14 from daniel-widrick/Szabka

Szabka

summakore committed:
Fix Major Typo

Costura.Fody targetFramework="net46" forgot the six.

daniel-widrick committed:
Merge pull request #15 from summakore/patch-1

Fix Major Typo

daniel-widrick committed:
Basic Preliminary Hiding

daniel-widrick committed:
Change process name base

undorev committed:
Bulgarian language added

daniel-widrick committed:
Merge pull request #19 from NaNraptor/CSharp

Bulgarian language added

daniel-widrick committed:
Add some randomness to certain behaviors

daniel-widrick committed:
preliminary

UltimateFishbot committed:
Merge pull request #126 from NaNraptor/CSharp

Bulgarian language added

daniel-widrick committed:
Preliminary edit for other language support

daniel-widrick committed:
Fix start stop key

daniel-widrick committed:
Merge pull request #20 from daniel-widrick/OtherLanguageSupport

Other language support

daniel-widrick committed:
Merge pull request #21 from daniel-widrick/window-title-random

Window title random

daniel-widrick committed:
title change

daniel-widrick committed:
Merge pull request #22 from daniel-widrick/window-title-random

title change

undorev committed:
Fix spelling mistakes

undorev committed:
Rename Bulgarian.xml to Български.xml

undorev committed:
Fixed spelling mistakes

Made it more readable

undorev committed:
updated new lines

fix of last commit

undorev committed:
Fix crash when WoW window not found, Add random delays, fix spelling, fix bulgarian

You will now notice that in the settings screen there are a couple more boxes, this is because the bot now expects you to put in a range, instead of a single hard coded value

Fixed the unhandled exception caused by WoW Window pointer being a "new IntPtr()"

Fixed a lot of english spellin gmistakes in Readme, and in English.xml

Fixed some bulgarian layout issues

undorev committed:
Merge branch 'CSharp' into CSharp

undorev committed:
Merge pull request #129 from NaNraptor/CSharp

Major update pull request, please look at @daniel-widrick 's fork and mine for more information

undorev committed:
updated to add @NaNraptor and latest changes

Lunat1q committed:
Refactor and some improvements (#131)

* Refactor + some improvements

* translation refactor

* extension

* ext

* namespace

* minor fixes

* comparasion change

undorev committed:
Update README.md

undorev committed:
Added chinese wow window name

undorev committed:
Merge branch 'CSharp' of https://github.com/UltimateFishbot/UltimateFishbot into CSharp

Crystalleaf committed:
Add Chinese Translation (#146)

* Create Chinese.xml

* Update Chinese.xml

* Update Chinese.xml

undorev committed:
make title case INcensitive

Spherikal commented:
Would it be possible to use a random timer instead of a set one? Or use a interval like /T 600-800 and the program will close anytime between 600 and 800 seconds or is that just not possible using *.bat files?

UltimateFishbot committed:
Update README.md

I've had a handful of people e-mail about bans now; who all insist they weren't doing anything suspicious, and I know there has been some chatting about it here as well.  I think it makes sense to emphasize that it is not risk free.
